llvm/test/Verifier/denormal-fp-math.ll

```diff
@@ -1,5 +1,6 @@
 ; RUN: not llvm-as %s -o /dev/null 2>&1 | FileCheck %s
 
+; CHECK-NOT: invalid value for 'denormal-fp-math' attribute
```

Comment:
Too fragile negative check on exact string. Use implicit-check-not for the broadest error check. Or just delete the valid case, verifier tests usually don't include them (those go in test/Assembler)